### PR TITLE
Configured prettier & eslint

### PR DESCRIPTION
**Pull request by msmorawski**
- Installed and configured Prettier & eslint
- Since now linter is run on pre-commit hook
- Solved errors detected by linter
- Formatted code according to the prettier rules
- Moved some of the packages to devDependencies

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -64,7 +52,38 @@
   },
   "husky": {
     "hooks": {
+      "pre-commit": "pretty-quick --staged && yarn lint",
       "pre-push": "yarn run test:cy -- --env configFile=local"
```
Comment (KedziaPawel) [inline package.json]: it's not a comment to this PR but to not forget about it, imho pre-push should use start-server-and-test https://docs.cypress.io/guides/guides/continuous-integration.html#Basics so the tests can be run without stating manually the app. This command would be also useful for CI

File: .eslintignore [hunk 2]
```diff
@@ -0,0 +1,5 @@
+node_modules
+build
+public
+cypress
```
Comment (KedziaPawel) [inline .eslintignore]: I think that we want to have eslint in cypress

File: .prettierignore [hunk 3]
```diff
@@ -0,0 +1,5 @@
+node_modules
+build
+public
+cypress
```
Comment (KedziaPawel) [inline .prettierignore]: same

File: .eslintignore (hunk 2 above)
Comment (msmorawski) [reply .eslintignore]: the that we can see cypress is here is not the "source code" of end-to-end tests but a folder with artifacts. But yea, it's confusing, I'll change the path of the test artifacts directory to not confuse

File: package.json (hunk 1 above)
Comment (msmorawski) [reply package.json]: Done

File: .eslintignore (hunk 2 above)
Comment (msmorawski) [reply .eslintignore]: Done

File: .prettierignore (hunk 3 above)
Comment (msmorawski) [reply .prettierignore]: Done

**Review by KedziaPawel — APPROVED** [review]

**Pull request merged**